### ISSUE TITLE
client.close() does not clear producer intervals

### ISSUE THREAD

**Issue by postypython** [nodejitsu/godot#29]
Hello,
first of all I wish to thank you for this great module: I have been using it since its publication.
Unfortunately I have the following issue while closing a udp client: it seems that  producer intervals are not cleared preventing server closing.
I am creating a new client and it's producer as per code below:

``` javacript
var godotProducer = godot.producer({
        host: '127.0.0.1',
        service: 'heartbeat',
        ttl: 1000 * 15,
}),

var godotClient = godot.createClient({
        type: 'udp',        
});

godotClient.add(godotProducer); 

godotClient.connect(10002); 
```

Then, if I do not run the following clearInterval instruction before closing godot's client connection, my tcp server won't close:

``` javacript
clearInterval(godotProducer.ttlId);

godotClient.close();
```

Many thanks in advance

**Comment by jcrugzz** [comment]
There is a small bug in the current npm package. This will be fixed when the changes in master are published. In the mean time you can use the master branch.

**Comment by indexzero** [comment]
Published fix in `godot@0.2.7`
